spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] I think it would be great if you add tests to your models and views since it is one of the requirements to add tests to your project so kindly add some tests for it 💯✨🤩💪🏻

![image](https://user-images.githubusercontent.com/58553711/231632171-7b183677-1584-4ac6-8a6a-2ac7147f1c15.png)